<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiStructDetectorDialog</class>
 <widget class="QDialog" name="uiStructDetectorDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>143</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="hl_Form">
     <item>
      <layout class="QFormLayout" name="fl_Detector">
       <item row="0" column="0">
        <widget class="QLabel" name="lb_Name">
         <property name="text">
          <string>Название:</string>
         </property>
         <property name="buddy">
          <cstring>ed_Name</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="ed_Name"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lb_Object">
         <property name="text">
          <string>Объект:</string>
         </property>
         <property name="buddy">
          <cstring>ed_Object</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="ed_Object"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lb_Direction">
         <property name="text">
          <string>Направление:</string>
         </property>
         <property name="buddy">
          <cstring>ed_Direction</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="ed_Direction">
         <item>
          <property name="text">
           <string>Въезд</string>
          </property>
          <property name="icon">
           <iconset resource="../../res.qrc">
            <normaloff>:/Icons/icons/import.ico</normaloff>:/Icons/icons/import.ico</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Выезд</string>
          </property>
          <property name="icon">
           <iconset resource="../../res.qrc">
            <normaloff>:/Icons/icons/export.ico</normaloff>:/Icons/icons/export.ico</iconset>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>uiStructDetectorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>uiStructDetectorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
